### DISCUSSION TITLE
host  Tandoor Recipes on a webserver on virtualmin

### DISCUSSION THREAD

**Discussion by manojknaikade** [TandoorRecipes/recipes#722] [Q&A]
Is there any way to host  Tandoor Recipes using virtualmin?

**Comment by vabene1111** [comment]
I actually have never heard of that. Do you have a link to a website or starter tutorial or can tell me a little more about it.

**Reply by manojknaikade** [reply]
virtualmin is basically a  open source web hosting control panel like plesk or c-panel. I am using it host my website on a old pc with ubuntu server running on it. I would like to host Tandoor Recipes on one of the subdomain on my website.

**Reply by vabene1111** [reply]
This is probably possible but I do not know what needs to be done to make an application installable on thi solattform.

Feel free to open an issue and I will look into this (although probably not with a super high priority)

**Reply by manojknaikade** [reply]
thank you @vabene1111, will do that.
